alibi/explainers/anchor_image.py

```diff
@@ -65,7 +65,7 @@ def __init__(
         Parameters
         ----------
         predictor
-            A callable that takes a tensor of N data points as inputs and returns N outputs.
+            A callable that takes a tensor of `N` data points as inputs and returns `N` outputs.
```

Comment:
Would be good to replace `tensor` with `array` everywhere we expect numpy arrays and reserve `tensor` for torch/tensorflow.